main.go

```diff
@@ -493,14 +471,14 @@ func (a *autographer) addSigners(signerConfs []signer.Configuration) error {
 		}
 		sids[signerConf.ID] = true
 		var (
-			s           signer.Signer
-			statsClient *signer.StatsClient
-			err         error
+			s signer.Signer
+			// statsClient *signer.StatsClient
```

Comment:
```suggestion
```

Reply:
(suggestion) Add TODO here to add metrics on config. [AUT-107](https://mozilla-hub.atlassian.net/browse/AUT-170) is probably the appropriate ticket.

Reply:
Missed another comment here.